packages/ember-views/lib/system/event_dispatcher.js

```diff
@@ -360,6 +360,10 @@ export default EmberObject.extend({
   },
 
   destroy() {
+    if (this.isDestroyed) {
```

Comment:
Are we calling `destroy` multiple times?

Reply:
yep, was thing of adding warning to `destroy` for calling multiple times what do you think ?